markdown/pxf/HDFSWritablePXF.html.md.erb

```diff
@@ -413,4 +411,4 @@ gpadmin=# SELECT * FROM read_pxf_tbl_seqwrit_RECKEY;
 (2 rows)
 ```
 
-The `recordkey` is displayed as `0`; a record key was not identified when you inserted entries in the writable table.
+The `recordkey` is displayed as `0`; you did not identify a record key when you inserted entries in the writable table.
```

Comment:
Edit:  The `recordkey` value is `0` because you did not identify a record key when you inserted entries into the writable table.